src/lib/LBT/LBT.cpp

```diff
@@ -112,6 +112,9 @@ void ICACHE_RAM_ATTR SetClearChannelAssessmentTime(void)
 
 SX12XX_Radio_Number_t ICACHE_RAM_ATTR ChannelIsClear(SX12XX_Radio_Number_t radioNumber)
 {
+  if (radioNumber == SX12XX_Radio_NONE)
+    return SX12XX_Radio_NONE;
```

Comment:
Not necessary, but also doesn't hurt. NONE should just fall through with no assessment being performed.

Reply:
Ah interesting I didn't think that the CCA would not be performed at all but I should have read further down because you're right.

What I was concerned about was this though where [it increments the LBTSuccessCalc period count](https://github.com/ExpressLRS/ExpressLRS/blob/3.x.x-maintenance/src/lib/LBT/LBT.cpp#L115) but has no chance of incrementing the value, so 100% LBTSuccessCalc LQ always comes out 99% for the NONE radio. This means the LBTSuccessCalc will always be 0%, so I believe it is necessary to prevent that, do you agree?

oh glob you just made me realize we have another bug in dynpower because LBT there is the denominator so if LBT's success count is 0 the TX will ... crash? I dunno what happens on a divide by zero on ESP32.

Reply:
Falling through would increase the LBTSuccess value which I think is what Cap wanted to avoid.

Reply:
> oh glob you just made me realize we have another bug in dynpower because LBT there is the denominator so if LBT's success count is 0 the TX will ... crash? I dunno what happens on a divide by zero on ESP32.

It throws an exception and reboots the module. Not what we want.